AtReconstruction/AtFilter/AtSCACorrect.h

```diff
@@ -23,24 +23,34 @@ class AtSCACorrect : public AtFilter {
    using RawEventPtr = std::unique_ptr<AtRawEvent>;
    AtMapPtr fMap;
 
-   RawEventPtr fBaseline;
-   RawEventPtr fPhase;
+   RawEventPtr fRawEvent;
+
+   Bool_t fDoBaseline;
+   Bool_t fDoPhase;
+   Bool_t fUseChanZero;
+
+   TString fBaseAugName;
+   TString fPhaseAugName;
 
 public:
-   AtSCACorrect(AtMapPtr map, TString baselineFilename, TString baselineEventName, TString phaseFilename,
-                TString phaseEventName);
-   AtSCACorrect(AtMapPtr map, RawEventPtr baseline, RawEventPtr phase);
+   AtSCACorrect(AtMapPtr map, TString filename, TString eventName, TString baselineAug, TString phaseAug);
+   AtSCACorrect(AtMapPtr map, RawEventPtr rawEvent, TString baselineAug, TString phaseAug);
```

Comment:
Add doxygen comment explaining the constructors and use of the filter. 

You're the only one using this method correct? This looks like a breaking change to the AtSCACorrect class. Anyone using the method before would have to regenerate the baseline data?

Reply:
I'm sure I was the only one using this method so at worst it breaks things for me. I will add documentation so others may use it if they wish.